pyhap/accessory.py

```diff
@@ -183,6 +183,17 @@ def pincode(self):
             self._pincode = util.generate_pincode()
         return self._pincode
 
+    def get_accessory_information(self):
```

Comment:
Okay I get your idea. Can we then call this `create_accessory_information` or something similar? My issues is that `get` intuitively will mean "get me the current accessory information", whereas it should be "create my custom accessory information". Does it make sense?